src/app/pages/directory-service/components/directory-services-config-form/trusted-domains-config/trusted-domains-config.component.ts

```diff
@@ -0,0 +1,206 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  input,
+  output,
+  OnInit,
+} from '@angular/core';
+import {
+  FormArray,
+  FormBuilder,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateModule } from '@ngx-translate/core';
+import { Observable, of } from 'rxjs';
+import { ActiveDirectorySchemaMode } from 'app/enums/directory-services.enum';
+import { Option } from 'app/interfaces/option.interface';
+import { IxCheckboxComponent } from 'app/modules/forms/ix-forms/components/ix-checkbox/ix-checkbox.component';
+import { IxFieldsetComponent } from 'app/modules/forms/ix-forms/components/ix-fieldset/ix-fieldset.component';
+import { IxInputComponent } from 'app/modules/forms/ix-forms/components/ix-input/ix-input.component';
+import { IxListItemComponent } from 'app/modules/forms/ix-forms/components/ix-list/ix-list-item/ix-list-item.component';
+import { IxListComponent } from 'app/modules/forms/ix-forms/components/ix-list/ix-list.component';
+import { IxSelectComponent } from 'app/modules/forms/ix-forms/components/ix-select/ix-select.component';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-trusted-domains-config',
+  templateUrl: './trusted-domains-config.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [
+    ReactiveFormsModule,
+    IxFieldsetComponent,
+    IxInputComponent,
+    IxSelectComponent,
+    IxCheckboxComponent,
+    IxListComponent,
+    IxListItemComponent,
+    TranslateModule,
+  ],
+})
+export class TrustedDomainsConfigComponent implements OnInit {
+  readonly trustedDomains = input<Record<string, unknown>[]>([]);
+  readonly trustedDomainsChanged = output<Record<string, unknown>[]>();
```

Comment:
You should be able to just pass in formControl from the parent directly and avoid changed output and separate form in this component.